lib/selects.bzl

```diff
@@ -107,6 +107,7 @@ def _config_setting_group(name, match_any = [], match_all = []):
       match_all: A list of `config_settings`. This group matches if *every*
           member in the list matches. If this is set, `match_any` must be not
           set.
+      visibility: Visibiliity of the main alias used.
```

Comment:
From a user perspective, would it be clearer to call this the visibility of the config_setting_group?

Also pedantic typo nit: double i's in the second visibility!

Reply:
typos and comment fixed